docs/user/quickstart.rst

```diff
@@ -0,0 +1,318 @@
+Deploying OpenWISP Using Ansible
+================================
+
+.. note::
+
+    It is recommended to use this procedure on clean virtual machines or
+    linux containers.
```

Comment:
move this note box  below and let's add another note here which mentions and links the section of the development version, eg: "If you are looking to use the latest features of OpenWISP, refer to ...".